osbrain/proxy.py

```diff
@@ -287,6 +311,8 @@ def proxy(self, name, timeout=3.):
             Proxy name, as registered in the name server.
         timeout : float
             Timeout, in seconds, to wait until the agent is discovered.
+        running : bool
+            Whether to wait for the agent to be running or not.
```

Comment:
Any particular reason you changed the default behavior in this method?

I mean, why do you think `Proxy()` should behave differently than `ns.proxy()` by default?

Reply:
Same as above,  I forgot to remove this part.